streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -3167,7 +3167,7 @@ object ZStream extends ZStreamPlatformSpecificConstructors with Serializable {
    * Constructs a stream from a range of integers (inclusive).
```

Comment:
Change comment from
```
Constructs a stream from a range of integers (inclusive)
```
to suggested
```
Constructs a stream from a range of integers (lower bound included, upper bound not included)
```

Reply:
Good one. It's good to make that clear, I will update